src/lightly_train/_methods/distillation/distillation.py

```diff
@@ -246,6 +250,36 @@ def trainable_modules(self) -> TrainableModules:
             modules=[self.student_embedding_model, self.student_projection_head]
         )
 
+    def configure_optimizers(self) -> OptimizerLRScheduler:
```

Comment:
Could we add a `lr_scale_method = "linear" | "sqrt"` argument to the optimizer args and handle the scaling in `Method.configure_optimizers` instead of copying the `configure_optimizers` for all method implementations?